test/epoch_token_locker.js

```diff
@@ -45,7 +45,7 @@ contract("EpochTokenLocker", async (accounts) => {
       await epochTokenLocker.deposit(ERC20.address, 100)
       await epochTokenLocker.deposit(ERC20.address, 100)
 
-      assert.equal(await epochTokenLocker.getPendingDepositAmount(user_1, ERC20.address), 200)
+      assert.equal((await epochTokenLocker.getPendingDepositAmount(user_1, ERC20.address)).toNumber(), 200)
```

Comment:
How did this change anything?

As far as I can tell the test failed because `<BN: 64>` (0x64 = 100) was not equal to 200. Do we have a flaky test?

Reply:
Goods question...